javaparser-core/src/main/java/com/github/javaparser/ast/nodeTypes/NodeWithOptionalScope.java

```diff
@@ -27,22 +27,11 @@
 import java.util.Optional;
 
 /**
- * Represents a node which can have a scope expression eg. method calls or
- * field accesses (object.method(), object.field).
+ * Represents a node which can have a scope expression eg. method calls (object.method()).
  */
 public interface NodeWithOptionalScope<N extends Node> {
 
     Optional<Expression> getScope();
 
     N setScope(Expression scope);
-
```

Comment:
My thinking was that they get generated anyway, but that's bad thinking now you mention it.